api/src/utils/cryptr.js

```diff
@@ -0,0 +1,33 @@
+// https://gist.github.com/vlucas/2bd40f62d20c1d49237a109d491974eb
+
+const crypto = require('crypto')
+
+class Cryptr {
```

Comment:
On a call with @kamicut we decided that this class would be a good candidate for a unit test (to make sure it's wrapping of the node crypto api is solid).